### PR TITLE
Added ability to use pasted JSON files for the GOOGLE_APPLICATION_CREDENTIALS environment variable

### PR DESCRIPTION
**Pull request by VinceFilary**

### CODE REVIEW

File: Sources/GoogleCloud/Common/OAuth/OAuthCredentialLoader.swift [hunk 1]
```diff
@@ -9,14 +9,25 @@ import Vapor
...
+        if let credentials = try? GoogleServiceAccountCredentials(json: credentialFilePath) {
+            return OAuthServiceAccount(credentials: credentials, scopes: config.scope, httpClient: client)
+        }
+        
         if let credentials = try? GoogleApplicationDefaultCredentials(contentsOfFile: credentialFilePath) {
             return OAuthApplicationDefault(credentials: credentials, httpClient: client)
         }
-
+        
+        if let credentials = try? GoogleApplicationDefaultCredentials(contentsOfFile: credentialFilePath) {
```
Comment (Andrewangeta) [inline Sources/GoogleCloud/Common/OAuth/OAuthCredentialLoader.swift]: @VinceFilary 
This is a duplicate call? Why is this needed? Did you mean to use the json API?

**Review by Andrewangeta — CHANGES_REQUESTED** [review]

File: Sources/GoogleCloud/Common/OAuth/OAuthCredentialLoader.swift (hunk 1 above)
Comment (VinceFilary) [reply Sources/GoogleCloud/Common/OAuth/OAuthCredentialLoader.swift]: Good catch! That is exactly what I meant. Copy and paste curse got the best of me!

**Review by Andrewangeta — APPROVED** [review]
